en/identity-server/7.0.0/docs/deploy/deployment-guide.md

```diff
@@ -64,7 +64,7 @@ The above recommendations can change based on the expected concurrency and perfo
 </tr>
 <tr class="even">
 <th>Java</th>
-<td>Oracle JDK 1.8</td>
+<td>Open JDK 11</td>
```

Comment:
Shall we just use JDK 11

On a different thought, shall we just link to the https://is.docs.wso2.com/en/next/deploy/environment-compatibility/#tested-operating-systems-and-jdks section similar to other entries in the table.